src/printer/tvmscript_printer.cc

```diff
@@ -416,26 +416,6 @@ class TVMScriptPrinter : public StmtFunctor<Doc(const Stmt&)>,
   }
 };
 
-/*!
- * \brief special method to print NDArray in TIR
- * \param arr the NDArray to be printed
- * \param os the output stream where the NDArray will be printed to
- */
-template <typename T>
-void NDArrayToTIR(::tvm::runtime::NDArray arr, std::ostream& os) {
```

Comment:
I removed this because otherwise it tries to write a huge constant array to a string. I don't think it is useful to print out the numerical value of constants.